faq.adoc

```diff
@@ -81,10 +81,10 @@ filter instead of `{organizationUuid}`
 [%collapsible]
 ====
 ****
-You can either use the `self` filter or call the following endpoint to fetch the organization UUID.
+You can use the `self` path parameter instead of specifying organization UUID when calling the APIs. Or, you can call the following endpoint to fetch the organization UUID.
```

Comment:
can you please change "path parameter" to "endpoint". Path parameter is name for a variable in an endpoint url not the url itself.

Reply:
Thanks! Fixed it.